<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page3_verificationcode</class>
 <widget class="QMainWindow" name="page3_verificationcode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>461</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="verification_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter the verification code:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="verification_lineEdit">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>70</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="next_pushButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>160</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>next</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_get_name">
     <property name="geometry">
      <rect>
       <x>-20</x>
       <y>50</y>
       <width>491</width>
       <height>221</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="pushButton_get_name">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>140</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>next</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_get_name">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>50</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_get_name">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter your name :</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
